velhron/dadata-bundle/1.0/post-install.txt

```diff
@@ -0,0 +1,8 @@
+<bg=blue;fg=white>              </>
```

Comment:
This file should be removed, otherwise it looks good to me 👍

Reply:
Deleted